src/code/z_bgcheck.c

```diff
@@ -4112,10 +4112,10 @@ s32 SurfaceType_IsIgnoredByProjectiles(CollisionContext* colCtx, CollisionPoly*
 }
 
 /**
- * CollisionPoly is conveyor enabled
- * Returns true if `poly` is a conveyor surface, else false
+ * CollisionPoly is a floor conveyor
+ * Returns true if `poly` is a surface that will push player while being stood on with a non-zero conveyor speed, else false
```

Comment:
Feel free to use more space for doc here, it's kind of unclear, it should probably mention the two behaviors